src/components/Label/Label.tsx

```diff
@@ -42,7 +38,7 @@ export interface Props {
   /**
    * The label text string
    */
-  text?: string | boolean;
```

Comment:
The text couldn't been a `bool`? How odd..

Reply:
Oh nice catch, thought I changed this to `string`, will change to `string`

Reply:
No you did; it's good now. I'm just confused about why they had it like that before 🤔